packages/components/src/components/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Label } from "../Label";
+import { Input } from "../Input";
+import { HelpText } from "../HelpText";
+import type { InputTypes } from "../Input";
+import { Box } from "../../primitives/Box";
+
+export interface FormInputProps
+  extends Omit<React.InputHTMLAttributes<HTMLInputElement>, "color" | "size"> {
```

Comment:
Shouldn't this be extending the props of our input component since `{...props}` is happening there?